File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/medicalroutinetreedto/d/Medicalroutinetreedto.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.medicalroutinetreedto.d
{
    /// <summary>
    /// Medicalroutinetreedto 的摘要说明。
    /// </summary>
    public class Medicalroutinetreedto : BaseDTO {

        public Medicalroutinetreedto() {
 
        }

        /// <summary>
        /// 主键
        /// </summary>
		public string Id {
            get { return getAttrVal<string>("Id",null); }
            set { setAttrVal<string>("Id", value); }
        }

        /// <summary>
        /// 父id
        /// </summary>
		public string Id_parent {
            get { return getAttrVal<string>("Id_parent",null); }
            set { setAttrVal<string>("Id_parent", value); }
        }

        /// <summary>
        /// 模板分类
        /// </summary>
		public string Id_ortmplca {
            get { return getAttrVal<string>("Id_ortmplca",null); }
            set { setAttrVal<string>("Id_ortmplca", value); }
        }

        /// <summary>
        /// 模板id
        /// </summary>
		public string Id_ortmpl {
            get { return getAttrVal<string>("Id_ortmpl",null); }
            set { setAttrVal<string>("Id_ortmpl", value); }
        }

        /// <summary>
        /// 名称
        /// </summary>
		public string Name {
            get { return getAttrVal<string>("Name",null); }
            set { setAttrVal<string>("Name", value); }
        }

        /// <summary>
        /// 描述
        /// </summary>
		public string Desc {
            get { return getAttrVal<string>("Desc",null); }
            set { setAttrVal<string>("Desc", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id", "Id_parent", "Id_ortmplca", "Id_ortmpl", "Name", "Desc"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.medicalroutinetreedto.d.Medicalroutinetreedto";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/medicalroutinetreedto/d/Medicalroutinetreedto.java
package iih.ci.ord.dto.medicalroutinetreedto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 医技常规模板树 DTO数据 
 * 
 */
public class Medicalroutinetreedto extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键
	 * @return String
	 */
	public String getId() {
		return ((String) getAttrVal("Id"));
	}
	/**
	 * 主键
	 * @param Id
	 */
	public void setId(String Id) {
		setAttrVal("Id", Id);
	}
	/**
	 * 父id
	 * @return String
	 */
	public String getId_parent() {
		return ((String) getAttrVal("Id_parent"));
	}
	/**
	 * 父id
	 * @param Id_parent
	 */
	public void setId_parent(String Id_parent) {
		setAttrVal("Id_parent", Id_parent);
	}
	/**
	 * 模板分类
	 * @return String
	 */
	public String getId_ortmplca() {
		return ((String) getAttrVal("Id_ortmplca"));
	}
	/**
	 * 模板分类
	 * @param Id_ortmplca
	 */
	public void setId_ortmplca(String Id_ortmplca) {
		setAttrVal("Id_ortmplca", Id_ortmplca);
	}
	/**
	 * 模板id
	 * @return String
	 */
	public String getId_ortmpl() {
		return ((String) getAttrVal("Id_ortmpl"));
	}
	/**
	 * 模板id
	 * @param Id_ortmpl
	 */
	public void setId_ortmpl(String Id_ortmpl) {
		setAttrVal("Id_ortmpl", Id_ortmpl);
	}
	/**
	 * 名称
	 * @return String
	 */
	public String getName() {
		return ((String) getAttrVal("Name"));
	}
	/**
	 * 名称
	 * @param Name
	 */
	public void setName(String Name) {
		setAttrVal("Name", Name);
	}
	/**
	 * 描述
	 * @return String
	 */
	public String getDesc() {
		return ((String) getAttrVal("Desc"));
	}
	/**
	 * 描述
	 * @param Desc
	 */
	public void setDesc(String Desc) {
		setAttrVal("Desc", Desc);
	}
}